components/new-contribution-flow/index.js

```diff
@@ -551,5 +628,7 @@ const addConfirmOrderMutation = graphql(
 );
 
 export default injectIntl(
-  withUser(addSignupMutation(addConfirmOrderMutation(addCreateOrderMutation(ContributionFlow)))),
+  withUser(
+    addSignupMutation(addConfirmOrderMutation(addCreateOrderMutation(addCreateCollectiveMutation(ContributionFlow)))),
```

Comment:
`addCreateCollectiveMutation` is using GQLV1, we should switch to V2 (eventually in a separate PR)

Reply:
Noted! I'll add a card.